crates/turborepo-lockfiles/src/npm.rs

```diff
@@ -135,6 +138,16 @@ impl Lockfile for NpmLockfile {
     fn encode(&self) -> Result<Vec<u8>, crate::Error> {
         Ok(serde_json::to_vec_pretty(&self)?)
     }
+
+    fn global_change_key(&self) -> Vec<u8> {
+        let mut buf = vec![b'n', b'p', b'm', 0];
+        {
+            let mut writer = BufWriter::new(&mut buf);
```

Comment:
I don't think `BufWriter` gets us anything here since we aren't wrapping up any I/O:
> BufWriter<W> can improve the speed of programs that make small and repeated write calls to the same file or network socket. It does not help when writing very large amounts at once, or writing just one or a few times. It also provides no advantage when writing to a destination that is in memory, like a [Vec](https://doc.rust-lang.org/std/vec/struct.Vec.html)<u8>.